acme/resource_acme_certificate.go

```diff
@@ -220,6 +220,12 @@ func resourceACMECertificateV5() *schema.Resource {
 				Default:   "",
 				Sensitive: true,
 			},
+			"revoke_certificate_on_destroy": {
+				Type:      schema.TypeBool,
+				Optional:  true,
+				Default:   true,
+				Sensitive: false,
```

Comment:
Nit: This does not need to be defined if it's the zero value (generally you only need to mark non-zero values in the TF schema as defined). I'll fix this when I add the tests.